### DISCUSSION TITLE
Manual ROI ohne Nummerierung wird falsch interpretiert

### DISCUSSION THREAD

**Discussion by Pergamon77** [jomjol/AI-on-the-edge-device#2506] [Q&A] [answered]
Hallo Zusammen,

ich habe alles soweit erfolgreich eingerichtet und es wird auch weitesgehend korrekt ausgelesen. Nur habe ich einen (von 4) manuellen ROIs als Nachkommastellen.
Leider hat einer von diesen ROIs keine Nummern, vermutlich deshalb funktioniert auch die Erfassung nicht korrekt. Zudem ist er bei der Draufsicht auch noch ein wenig verdeckt.
Wie kann ich hier Abhilfe schaffen (ohne selbst Zahlen draufzuschreiben) ?

Dankeschön!
![image](https://github.com/jomjol/AI-on-the-edge-device/assets/128689716/f34c5bd5-b20b-4d38-9c5a-ab1d3d43a37c)

**Comment by caco3** [comment, marked as answer]
Die Zahlen sollten eigentlich nicht relevant sein, es wird der Winkel des Zeigers verwendet.

Wenns aber nur bei dem Zeige ein Problem ist, ist es am besten, Du stellst uns Trainingsdaten bereit, siehe https://jomjol.github.io/AI-on-the-edge-device-docs/collect-new-images/#collect-images-to-improve-the-models

**Comment by Pergamon77** [comment]
Achso, danke für den Hinweis.

Zwischenzeitlich habe ich mal das andere Model für die analogen ROIs ausprobiert. Bislang scheint es damit zu funktionieren.
